vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topDiskAvgLatencyHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top avg disk latency by ESXi hosts',
+          targets=[t.topDiskAvgLatencyHosts],
+          description='The ESXi hosts with the highest average disk latency in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topPacketErrorRateHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top packet errors by ESXi hosts',
+          targets=[t.topPacketErrorRateHosts],
+          description='The ESXi hosts with the highest percentage of packet errors in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      hostCPUUsage:
+        commonlib.panels.generic.timeSeries.base.new(
+          'CPU usage',
+          targets=[t.hostCPUUsage],
+          description='The amount of CPU used by the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('rotmhz')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostCPUUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'CPU utilization',
+          targets=[t.hostCPUUtilization],
+          description='The CPU utilization percentage of the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostMemoryUsage:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Memory usage',
+          targets=[t.hostMemoryUsage],
+          description='The amount of memory used by the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostMemoryUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'Memory utilization',
+          targets=[t.hostMemoryUtilization],
+          description='The memory utilization percentage of the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostModifiedMemory:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Modified memory',
+          targets=[t.hostModifiedMemoryBallooned, t.hostModifiedMemorySwapped],
+          description='The amount of memory that has been swapped or ballooned on the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.options.legend.withPlacement('right'),
```

Comment:
There's an issue with the legend on this panel, where some entries begin with `-`, which I reckon is due to a missing cluster label?
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/029300b7-91af-4f48-8d1e-0414cac04ffc)

Reply:
@Dasomeone these are actually standalone hosts (won't have a cluster). I don't love this legend either....but I couldn't figure out if there was some "conditional" that you could do here.

Reply:
It's not possible based on metric inspection, no, but in this case I think it might be appropriate if we simply move the cluster part to be appended rather than prefixed?

Not perfect, but would be a little cleaner.

Alternatively, we could drop the cluster part from the legend entirely and rely on the selector dropdowns